src/components/AppElements/components/CodeEditor.vue

```diff
@@ -339,7 +335,10 @@ export default {
 		},
 		// Html import
 		htmlImportCode() {
-			return htmlImportCodeGenerator();
+			return htmlImportCodeGenerator(this.getImportSettings);
+		},
+		htmlImportHooks() {
```

Comment:
move this computed under the //hooks comment

Reply:
nono